tools/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.7",
+  "PreviewReleaseTag": "v7.1.0-rc.1",
   "ServicingReleaseTag": "v6.2.7",
```

Comment:
@TravisEz13 Is this the right change for `metadata.json` for a RC release?

Reply:
preview should stay preview